docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+## Deploy the TLS CA
+
+Regardless of whether you are setting up a TLS CA, an organization CA or an intermediate CA, the process follows the same overall steps. The differences will be in the modifications you make to the CA server configuration .yaml file. The following steps provide an overview of the process:
+
+- [Step one: Initialize the CA server](#initialize-the-tls-ca-server)
+- [Step two: Modify the CA server configuration](#modify-the-tls-ca-server-configuration)
+- [Step three: Delete the CA server certificates](#delete-the-tls-ca-server-certificates)
+- [Step four: Start the CA server](#start-the-tls-ca-server)
+- [Step five: Enroll bootstrap user with TLS CA](#enroll-node-admins-with-tls-ca)
+
+When you deploy any node, you have three options for your TLS configuration:
+- No TLS. _Not recommended for a production network._
+- Server-side TLS.
+- Mutual TLS.
+
+This process will configure a CA with server-side TLS enabled which is required in order to use the Fabric [Service Discovery](../discovery-overview.html) and [Private Data](../private-data-arch.html) features. Mutual TLS is disabled by default. If you need to use mutual TLS, refer to the [TLS configuration settings](ca-config.html#tls).
+
+### Before you begin
+
+You should have already downloaded and copied the Fabric CA server binary `fabric-ca-server` to a clean directory on your machine. For purposes of these instructions, we put the binary in its own folder named `fabric-ca-server-tls`.
+
+  ```
+  mkdir fabric-ca-server-tls
+  ```
+  Copy the `fabric-ca-server` binary into this folder.
+
+### Initialize the TLS CA server
+
+The first step to deploy a CA server is to "initialize" it. Run the following CA server CLI command to initialize the server by specifying the admin user id and password for the CA:
+
+```
+./fabric-ca-server init -b <ADMIN_USER>:<ADMIN_PWD>
+```
+
+For example:
+```
+cd fabric-ca-server-tls
+./fabric-ca-server init -b tls-admin:tls-adminpw
+```
+The `-b` flag bootstraps the admin username and password to the CA server which effectively "registers" the CA admin user with the server for you, so an explicit Fabric CA client CLI `register` command is not required for the bootstrapped user. All CA users need to be "registered" and then "enrolled" with the CA, except for this CA admin identity which is implicitly registered by using the `-b` flag.
+
+**Note: This example is for illustration purposes only. Obviously, in a production environment you would never use `tls-admin` and `tls-adminpw` as the bootstrap username and password.**  Be sure that you record the admin id and password that you specify. They are required later when you issue enroll commands against the CA. It can help to use a meaningful id to differentiate which server you are transacting with and follow secure password practices.  
```

Comment:
"They are required later when you issue enroll commands"...
do you mean register commands?  or maybe register and enroll commands?

Reply:
this comment is still outstanding.

Reply:
good point. I left out register. Added it now.